front/components/Button/button.spec.tsx

```diff
@@ -0,0 +1,24 @@
+import { render, screen } from '@testing-library/react';
+
+import Button from './index';
+
+test('Button', () => {
+  render(<Button>Hello</Button>);
+
+  const button = screen.getByRole('button');
+  expect(button).toHaveClass('MuiButton-text');
```

Comment:
This looks like an implementation detail. I don't think there is anything to test for this component.